docs/csharp/tutorials/console-webapiclient.md

```diff
@@ -43,21 +43,17 @@ comfortable with.
 The first step is to create a new application. Open a command prompt and
 create a new directory for your application. Make that the current
 directory. Type the command `dotnet new console` at the command prompt. This
-creates the starter files for a basic "Hello World" application.
-
-Before you start making modifications, let’s go through the steps to run
-the simple Hello World application. After creating the application, type
-`dotnet restore` ([see note](#dotnet-restore-note)) at the command prompt. This command runs the NuGet
-package restore process. NuGet is a .NET package manager. This command
-downloads any of the missing dependencies for your project. As this is a
+creates the starter files for a basic "Hello World" application. As this is a
 new project, none of the dependencies are in place, so the first run will
-download the .NET Core framework. After this initial step, you will only
-need to run `dotnet restore` ([see note](#dotnet-restore-note)) when you add new dependent packages, or update
-the versions of any of your dependencies.
+download the .NET Core framework, install a development certificate and run
+the NuGet package manager to restore missing dependencies.
 
-After restoring packages, you run `dotnet build`. This executes the build
-engine and creates your application. Finally, you execute `dotnet run` to
-run your application.
+Before you start making modifications, type
+`dotnet run` ([see note](#dotnet-restore-note)) at the command prompt to
+run your application. If there are missing dependencies this command will also
```

Comment:
I suggest a different wording for the sentence starting with "If there are missing....":

> `dotnet run` automatically performs `dotnet restore` if your environment is missing dependencies. It also performs `dotnet build` if your application needs to be rebuilt. 

That would clarify that `dotnet build` and `dotnet restore` are performed under different conditions.

Reply:
@BillWagner Will do...